### DISCUSSION TITLE
No shadows for a specific theme

### DISCUSSION THREAD

**Discussion by kagiorgis** [saadeghi/daisyui#1448] [Q&A] [answered]
In my **_input.css_** I have a "**_parent-box_**" that has "**_shadow-md_**" and for the _**Lofi**_ theme I want this shadow to **not** be visible. 

Any ideas on how to do this?

**Comment by saadeghi** [comment, marked as answer]
Assuming you have this on your CSS file:
```css
.parent-box{
  @apply shadow-md;
}
```
**Method 1:**  
You can extend and edit `lofi` theme on your `tailwind.config.js` file like this:  
(If you need other themes too, you should add them to the `themes` array)
```js
module.exports = {
  //...

  daisyui: {
    themes: [
      "light",
      {
        lofi: {
          ...require("daisyui/src/colors/themes")["[data-theme=lofi]"],
          ".parent-box": {
            "box-shadow": "none",
          }
        },
      },
    ],
  },

}
```
This is how you you can add any style to a specific theme.  
Example: https://play.tailwindcss.com/n00teeAjOu?file=config

**Method 2:**  
You can do the same thing in CSS:  
```css
[data-theme=lofi] .parent-box {
  box-shadow: none;
}
```
Example: https://play.tailwindcss.com/tUcTvbWJQl?file=css

**Reply by kagiorgis** [reply]
Exactly what I was looking for, brilliant, thank you so much! I was messing around the config file customising themes, but I had no idea how you could customise specific .rules like this! Cheers!
